README.md

```diff
@@ -0,0 +1,73 @@
+# Student Robotics OS
+
+Student Robotics OS is a Linux distribution for use on the [Student Robotics](https://studentrobotics.org) kit.
+
+This repository contains build scripts to compile disk images and update bundles.
+
+## Setup Your Build Environment
+
+Firstly, you will need to install the dependencies for Yocto.
+
+On Ubuntu or Debian, you can install with the following command:
+
+```shell
+sudo apt install gawk wget git-core diffstat unzip texinfo \
+     build-essential chrpath socat cpio python python3 python3-pip python3-pexpect \
+     xz-utils lz4 debianutils iputils-ping libsdl1.2-dev xterm zstd bmap-tools
+```
+
+If you are using a x86-based computer, you will also need to install a cross-compiler:
+
+```shell
+sudo apt install gcc-multilib
+```
+
+Once your have installed the system dependencies, you will need to setup the repository:
+
+```shell
+git clone git@github.com:srobo/robot-image.git
+git submodule update --init --recursive
```

Comment:
You need to cd into the directory before running the submodule command

Reply:
Better yet `git clone --recursive`